src/blocks/cpu.rs

```diff
@@ -206,3 +213,18 @@ impl Block for Cpu {
         self.id
     }
 }
+
+/// Read the cpu turbo boost status from kernel sys interface
+/// or intel pstate interface
+fn boost_status() -> Result<bool> {
+    if let Ok(boost) = fs::read_to_string("/sys/devices/system/cpu/cpufreq/boost") {
+        return Ok(boost.starts_with("1"));
+    } else if let Ok(no_turbo) = fs::read_to_string("/sys/devices/system/cpu/intel_pstate/no_turbo")
+    {
+        return Ok(no_turbo.starts_with("0"));
+    }
+    Err(BlockError(
```

Comment:
Note that you can use `block_error()' function on `Resut` type instead of manually creating `BlockError'. The internals if `BlockError` will change at some point to include addition information.

Reply:
Yes, but I think, may be remove error at all? Because if boost_status return Err, that will break cpu block, even if no {boost} template used... Just return bool?

Reply:
Good point. Or perhaps `Option<bool>` and default to `""`?

Reply:
Done